conf/config.local.php

```diff
@@ -1,9 +1,886 @@
+ * pour faire une mise à jour en deux clics.
+ *
+ * Il est conseillé de désactiver cette fonctionnalité si vous ne voulez pas
+ * permettre à un utilisateur de casser l'installation !
+ *
+ * Si cette constante est désactivée, mais que ENABLE_TECH_DETAILS est activé,
+ * la vérification de nouvelle version se fera quand même, mais plutôt que de proposer
+ * la mise à jour, Paheko proposera de se rendre sur le site officiel pour
+ * télécharger la mise à jour.
+ *
+ * Défaut : true
+ *
+ * @var bool
+ */
+
+const ENABLE_UPGRADES = false;
+
+/**
+ * Utilisation de cron pour les tâches automatiques
+ *
+ * Si "true" on s'attend à ce qu'une tâche automatisée appelle
+ * les scripts suivants:
+ * - scripts/cron.php toutes les 24 heures (envoi des rappels de cotisation,
+ * création des sauvegardes)
+ * - scripts/emails.php toutes les 5 minutes environ (envoi des emails en attente)
+ *
+ * Si "false", les actions de scripts/cron.php seront effectuées quand une personne
+ * se connecte. Et les emails seront envoyés instantanément (ce qui peut ralentir ou
+ * planter si un message a beaucoup de destinataires).
+ *
+ * Défaut : false
+ */
+
+const USE_CRON = true;
+
+/**
+ * Activation de l'envoi de fichier directement par le serveur web.
+ * (X-SendFile)
+ *
+ * Permet d'améliorer la rapidité d'envoi des fichiers.
+ * Supporte les serveurs web suivants :
+ * - Apache avec mod_xsendfile (paquet libapache2-mod-xsendfile)
+ * - Lighttpd
+ *
+ * N'activer que si vous êtes sûr que le module est installé et activé (sinon
+ * les fichiers ne pourront être vus ou téléchargés).
+ * Nginx n'est PAS supporté, car X-Accel-Redirect ne peut gérer que des fichiers
+ * qui sont *dans* le document root du vhost, ce qui n'est pas le cas ici.
+ *
+ * Pour activer X-SendFile mettre dans la config du virtualhost de Paheko:
+ * XSendFile On
+ * XSendFilePath /var/www/paheko
+ *
+ * (remplacer le chemin par le répertoire racine de Paheko)
+ *
+ * Détails : https://tn123.org/mod_xsendfile/
+ *
+ * Défaut : false
+ */
+
+//const ENABLE_XSENDFILE = false;
+
+/**
+ * Serveur NTP utilisé pour les connexions avec TOTP
+ * (utilisé seulement si le code OTP fourni est faux)
+ *
+ * Désactiver (false) si vous êtes sûr que votre serveur est toujours à l'heure.
+ *
+ * Défaut : fr.pool.ntp.org
+ */
+
+//const NTP_SERVER = 'fr.pool.ntp.org';
+
+/**
+ * Désactiver l'envoi d'e-mails
+ *
+ * Si positionné à TRUE, l'envoi d'e-mail ne sera pas proposé, et il ne sera
+ * pas non plus possible de récupérer un mot de passe perdu.
+ * Les parties de l'interface relatives à l'envoi d'e-mail seront cachées.
+ *
+ * Ce réglage est utilisé pour la version autonome sous Windows, car Windows
+ * ne permet pas l'envoi d'e-mails.
+ *
+ * Défaut : false
+ * @var bool
+ */
+
+//const DISABLE_EMAIL = false;
+
+
+/**
+ * Hôte du serveur SMTP, mettre à false (défaut) pour utiliser la fonction
+ * mail() de PHP
+ *
+ * Défaut : false
+ */
+
+//const SMTP_HOST = false;
```

Comment:
(ideally we could configure email with the new system_user thingy)

Reply:
`allow_email` can't be provided if there is no user (no sso or yunohost LDAP), isn't it ?

Reply:
Why the line is on red ?
https://github.com/YunoHost-Apps/paheko_ynh/blob/78d240d74eb38eb5ef62d27d6d78fa901cbe7e65/manifest.toml#L46-L48